src/typing/index.ts

```diff
@@ -0,0 +1,80 @@
+export type TUser = {
+  data: TUserProps;
+};
```

Comment:
поле ```data```  кажется очень абстрактным и я задаюсь вопросом, а так ли оно нужно? Может ```TUser``` и ```TUserProps``` - это один и тот же тип? Если хочется разделить, то можно написать вот так:
```suggestion
export type TUser = TUser
```
Но я бы оставил что-то одно, чтобы не путаться